3-enrich/emr-etl-runner/config/config.yml.sample

```diff
@@ -25,7 +25,7 @@ aws:
         errors: ADD HERE     # Leave blank unless :continue_on_unexpected_error: set to true below
         archive: ADD HERE    # Where to archive shredded events to, e.g. s3://my-archive-bucket/shredded
   emr:
-    ami_version: 4.5.0
+    ami_version: 5.3.0      # Relational Database Shredder >= 0.12 and Spark Enrich >= 1.9.0 require >= 5.3.0
```

Comment:
never tested it with 5.5.0 since it wasn't out when I wrote all this but I see no reason why it wouldn't